bin/trim-shrinkwrap.js

```diff
@@ -1,28 +0,0 @@
-#!/usr/bin/env node
-
-/**
- * Removes the redundant fields on the npm-shrinkwrap.json file.
```

Comment:
Is this no longer an issue?

Reply:
That's my understanding. See https://jpospisil.com/2017/06/02/understanding-lock-files-in-npm-5.html

> You might have noticed that the resolved field is still present in the file pointing to a specific URI. Note however that NPM is now able to figure out (based on the settings in .npmrc) that the machine is configured to use a different registry and if so, it will transparently use it instead. This plays well with the integrity field because it now doesn’t matter from where the package came from as long as it matches the signature.